src/index.js

```diff
@@ -0,0 +1,67 @@
+import './style.css';
+// eslint-disable-next-line
```

Comment:
Please add an empty line before your comments, it will be enhancing the readability of your code, in other words, the readers will be able to differentiate what snippet is related.

Reply:
Hi, @robinbenitezmora thank you for your suggestions, they were very useful, I already solved the problem! 👍🏼